changelog/lambdacomp.dd

```diff
@@ -0,0 +1,17 @@
+Lambda comparison using __traits(isSame, ...)
+
+It is now possible to compare two lambda functions, under certain
+constraints, using __traits(isSame, lamda1, lambda2). In order to
+correctly compare two lambdas, the following conditions must be
+satisfied:
+
+-> the lambda function arguments must not have a template instantiation
+as an explicit argument type. Any other argument types (basic,
+user-defined, template) are supported.
+
+-> the lambda function body must contain a single expression (no return statement)
+which contains only numeric values, manifest constants, enum values and arguments.
+If the expression contains local variables, function calls or return statements,
+the function is considered uncomparable.
+
+These limitation will be lifted in the next release version.
```

Comment:
> These limitation will be lifted in the next release version.

I would recommend avoiding being this specific. There are not guarantees that those changes will make it into the next release. I would say something like: "These limitations might be lifted in the future".